explainer.md

```diff
@@ -1,751 +1,423 @@
-similarly named functions (e.g. `element-children()`).
-[Here](https://jsbin.com/bisoleziyi/edit?html,output) is a polyfill example of a
-single image mode transition built using the existing element() support in
-Firefox.
+```css
+::page-transition-container(foo) :>> outgoing-image {
+  /* … */
+}
+```
 
-Note that we are not currently planning on exposing the stand-alone method of
-capturing the elements, as would be possible with an `element()` function.
-Instead, the capturing and displaying the contents via the pseudo elements is an
-internal detail.
+This will play well with CSS nesting:
+
+```css
+::page-transition-container(foo) {
+  & :>> outgoing-image {
+    /* … */
+  }
+  & :>> incoming-image {
+    /* … */
+  }
+}
+```
+
+## Transitions targeted to a specific element
+
+In the current design, the transition acts across the whole document. However, developers have expressed interest in using this system, but limited to a single element. For example, allowing two independent components to perform transitions.
+
+This is being discussed in https://github.com/WICG/shared-element-transitions/issues/52.
 
 # Security/Privacy Considerations
 
 The security considerations below cover same-origin transitions.
 
-- Script can never read pixel content in the images. This is necessary since the
-  document may embed cross-origin content (iframes, CORS resources, etc.) and
-  multiple restricted user information (visited links history, dictionary used
-  for spell check, etc.)
-- If an element is captured as a 'computed style + content image', any external
-  resources specified on the container, such as background images, will be
-  re-fetched in the context of the new page to account for differences in
-  sandboxing.
+- Script can never read pixel content in the images. This is necessary since the document may embed cross-origin content (iframes, CORS resources, etc.) and multiple restricted user information (visited links history, dictionary used for spell check, etc.)
+- If an element is captured as a 'computed style + content image', any external resources specified on the container, such as background images, will be re-fetched in the context of the new page to account for differences in sandboxing.
 
-Cross-origin transitions aren't yet defined, but are likely to be heavily
-restricted.
+Cross-origin transitions aren't yet defined, but are likely to be heavily restricted.
 
 # Interactivity and accessibility
 
-Page transitions are a purely visual affordance. In terms of interactivity,
-transition elements will behave like `div`s regardless of the original element.
-
-Developers could break this intent by adding interactivity directly to the
-transition element, e.g. by deliberately adding a `tabindex` attribute. But this
-isn't recommended.
+Page transitions are a purely visual affordance. In terms of interactivity, transition elements will behave like `div`s regardless of the original element.
```

Comment:
nit: Could these 3 be written as points.

Reply:
Hmm, but "Developers could break this intent" seems like a follow-on statement, where as a bulleted list is usually for independent statements.

Reply:
I missed that. So 2 points. :) I put this in a follow up PR.